### PR TITLE
Add `.transiting` class

### PR DESCRIPTION
**Pull request by blasten**

### CODE REVIEW

File: paper-progress.html [hunk 1]
```diff
@@ -63,6 +63,12 @@
       height: 4px;
     }
 
+    :host(.transiting) #activeProgress,
+    :host(.transiting) #secondaryProgress {
+      -webkit-transition: -webkit-transform 0.08s ease;
+      transition: transform 0.08s ease;
```
Comment (frankiefu) [inline paper-progress.html]: How about adding a custom property here so users can override the transition?

Comment (blasten) [inline paper-progress.html]: :+1:

**Comment by blasten** [comment]
PTAL @frankiefu

**Comment by frankiefu** [comment]
LGTM :+1:

**Pull request merged**